src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
```

Comment:
`import` is the canonical way of only bringing only one name into the current namespace, so I would also prefer the old version.